src/citrine/resources/branch.py

```diff
@@ -142,9 +222,12 @@ def list(self, *, per_page: int = 20) -> Iterator[Branch]:
         Returns
         -------
         Iterator[Branch]
-            Active branches in this collection.
+            All branches in this collection.
 
         """
+        warnings.warn("Beginning in the 3.0 release, this method will only list unarchived "
```

Comment:
That it listed all branches conflicted with the documentation, and also doesn't match the approach taken in our other collections.